nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -27,11 +27,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <Target Name="CsWinRTRemoveWinMDReferences" BeforeTargets="ResolveReferences;BeforeCompile" AfterTargets="AfterResolveReferences">
     <ItemGroup>
       <!--Move winmd references into private item group to prevent subsequent winmd reference errors-->
-      <CsWinRTRemovedReferences Include="@(ReferencePath)" Condition="'%(ReferencePath.WinMDFile)' == 'true'" />
+      <CsWinRTRemovedReferences Include="@(ReferencePath)" Condition="'%(ReferencePath.Extension)' == '.winmd'" />

```

Comment:
Is ReferencePath an object with fields `WinMDFile` and `Extension`? Or did you change the `WinMDFile` field to `Extension`?